<?xml version="1.0" encoding="UTF-8" ?>
<launch>

    <node pkg="ris_calib" type="ris_calib_merge" name="ris_calib_merge_node" output="screen">
        <param name="config_path" value="$(find ris_calib)/config/config-merge.yaml"
               type="string"/>
    </node>

</launch>